### PR TITLE
[TEST] Add tensor clone test

### PR DESCRIPTION
**Pull request by gichan-jang**
- Add tensor clone test
ml_tensor_info_clone function changed to check the number of the
tensors of source info.
So, set the num_tesnors for source info.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

### CODE REVIEW

**Review by again4you — APPROVED** [review]
Nice catch!

File: tests/capi/unittest_capi_inference.cc [hunk 1]
```diff
@@ -2812,6 +2812,30 @@ TEST (nnstreamer_capi_util, info_get_tsize_03_n)
...
+
+  status = ml_tensors_info_create (&in_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+  status = ml_tensors_info_create (&out_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+  status = ml_tensors_info_set_count (in_info, 1);
+  EXPECT_EQ (status, ML_ERROR_NONE);
+
+  status = ml_tensors_info_clone (out_info, in_info);
+  EXPECT_EQ (status, ML_ERROR_NONE);
```
Comment (dongju-chae) [inline tests/capi/unittest_capi_inference.cc:2831]: How about adding extra test? like
```
  ml_tensors_info_get_count (out_info, &count);
  EXPECT_EQ (count, 1);
``

Comment (gichan-jang) [reply tests/capi/unittest_capi_inference.cc:2845]: Thanks! I needed to set type and dimension also. Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**